esp/src/eclwatch/nls/hpcc.js

```diff
@@ -196,10 +200,13 @@ define({root:
     Format: "Format",
     Forums: "Forums",
     Forward: "Forward",
+    FoundFile2: "Found File:",
+    FoundFileMessage: "A found file has all of its parts on disk that are not referenced in the Dali server. All the file parts are accounted for so they can be added back to the Dali server. They can also be deleted from the cluster, if required.",
```

Comment:
Does this text come from the legacy version?  @JamesDeFabia do you want some input on these messages?